app/Services/Phoenix/Phoenix.php

```diff
@@ -13,9 +13,9 @@ class Phoenix extends RestApiClient
      */
     public function __construct()
     {
-        $url = config('services.phoenix.uri') . '/api/' . config('services.phoenix.version') . '/';
+        $this->url = config('services.phoenix.uri') . '/api/' . config('services.phoenix.version') . '/';
```

Comment:
You don't have to store this on a property, since it gets set as the default base URI for the client. Setting it as a local variable `$url` and passing it along to the RestApiClient's constructor should be enough!